sources/packages/backend/apps/workers/src/controllers/supporting-user/_test_/e2e/supporting-user.controller.createIdentifiableSupportingUsers.e2e-spec.ts

```diff
@@ -0,0 +1,306 @@
+      undefined,
+      {
+        applicationData: {
+          workflowName: "some-workflow",
+          parents: [
+            {
+              parentFullName,
+            },
+          ],
+        } as ApplicationData,
+      },
+    );
+    const fakePayload = createFakeCreateIdentifiableSupportingUsersPayload({
+      applicationId: savedApplication.id,
+      supportingUserType: SupportingUserType.Parent,
+      parent: 1,
+      isAbleToReport: true,
+    });
+
+    // Act
+    const result =
+      await supportingUserController.createIdentifiableSupportingUsers(
+        createFakeWorkerJob<
+          CreateIdentifiableSupportingUsersJobInDTO,
+          ICustomHeaders,
+          CreateIdentifiableSupportingUsersJobOutDTO
+        >(fakePayload),
+      );
+
+    // Asserts
+    // Validate DB creation.
+    const updatedApplication = await db.application.findOne({
+      select: {
+        id: true,
+        supportingUsers: {
+          id: true,
+          supportingUserType: true,
+          isAbleToReport: true,
+          fullName: true,
+        },
+      },
+      relations: { supportingUsers: true },
+      where: {
+        id: savedApplication.id,
+      },
+    });
+    expect(updatedApplication.supportingUsers).toHaveLength(1);
+    const [parent] = updatedApplication.supportingUsers;
+    expect(parent).toEqual({
+      id: expect.any(Number),
+      supportingUserType: SupportingUserType.Parent,
+      isAbleToReport: true,
+      fullName: parentFullName,
+    });
+    // Validate job result.
+    expect(result).toEqual({
+      resultType: MockedZeebeJobResult.Complete,
+      outputVariables: {
+        createdSupportingUserId: parent.id,
+      },
+    });
+  });
+
+  it("Should create the second parent supporting user and save the associated full name when the two parent's information is added to the student application.", async () => {
+    // Arrange
+    const parentFullName1 = faker.datatype.uuid();
+    const parentFullName2 = faker.datatype.uuid();
+    const savedApplication = await saveFakeApplication(
+      db.dataSource,
+      undefined,
+      {
+        applicationData: {
+          workflowName: "some-workflow",
+          parents: [
+            {
+              parentFullName: parentFullName1,
+            },
+            {
+              parentFullName: parentFullName2,
+            },
+          ],
+        } as ApplicationData,
+      },
+    );
+    const fakePayload = createFakeCreateIdentifiableSupportingUsersPayload({
+      applicationId: savedApplication.id,
+      supportingUserType: SupportingUserType.Parent,
+      parent: 2,
+      isAbleToReport: false,
+    });
+
+    // Act
+    const result =
+      await supportingUserController.createIdentifiableSupportingUsers(
+        createFakeWorkerJob<
+          CreateIdentifiableSupportingUsersJobInDTO,
+          ICustomHeaders,
+          CreateIdentifiableSupportingUsersJobOutDTO
+        >(fakePayload),
+      );
+
+    // Asserts
```

Comment:
Minor. `// Assert` do we commonly use in plural form?

Reply:
Updated